tests/libcxx-backports/default_accessor/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+mdspan_add_test(default_accessor.access.pass)
+mdspan_add_test(default_accessor.ctor.conversion.pass)
+mdspan_add_test(default_accessor.ctor.default.pass)
+#mdspan_add_test(default_accessor.element_type.verify)
+mdspan_add_test(default_accessor.offset.pass)
+mdspan_add_test(default_accessor.types.pass)
```

Comment:
should we do a glob and detect the files ending in`. pass.cpp`?

Reply:
possibly.